.github/workflows/size-limit.yml

```diff
@@ -25,6 +26,18 @@ jobs:
         with:
           node-version: 16
           cache: yarn
+
+      - name: Restore cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            **/.eslintcache
+            **/.turbo
+            node_modules/.cache/turbo
+          key: ${{ runner.os }}-test-v1-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-test-v1-
+
```

Comment:
We can probably remove the [size-limit](https://github.com/Shopify/polaris/blob/ac19525082a944d9d858c5f02470abf82ab065db/package.json#L45) run-script now and update the `build_script` below e.g.
`build_script: build -- --filter=@shopify/polaris...`